test/prepare-manual-query.test.js

```diff
@@ -0,0 +1,16 @@
+var prepareManualQuery = require('../lib/prepare-manual-query'), assert = require('assert')
```

Comment:
This looks a little odd, although I imagine its like this because of the linting

It should look like

```js
var prepareManualQuery = require('../lib/prepare-manual-query')
  , assert = require('assert')
```